packages/table/src/tableBody.tsx

```diff
@@ -215,8 +215,24 @@ export class TableBody extends React.Component<ITableBodyProps, {}> {
             return undefined;
         }
 
-        const target = this.locateClick(e.nativeEvent as MouseEvent);
-        return renderBodyContextMenu(new MenuContext(target, selectedRegions, grid.numRows, grid.numCols));
+        const targetRegion = this.locateClick(e.nativeEvent as MouseEvent);
+        const { numRows, numCols } = grid;
+
+        const menuContext = new MenuContext(targetRegion, selectedRegions, numRows, numCols);
```

Comment:
Should we not be fixing the selectedRegions first here? Before we render the menu?

Reply:
Sure, fixed